README.md

```diff
@@ -1,2 +1,43 @@
-# Personal-Portfolio
```

Comment:
Consider adding a title of the project along with its description. Having a professional README file is a must for any project because it is going to be the face of your project for any visitor seeing your project.